src/apyfloat_wrapper.cc

```diff
@@ -623,5 +623,29 @@ void bind_float(nb::module_& m)
                 Quantization mode to use. If not provided, the global mode,
                 see :func:`get_float_quantization_mode`, is used.
             )pbdoc"
-        );
+        )
+        .def("next_up", &APyFloat::next_up, R"pbdoc(
+            Get the least floating-point number in the same format that compares greater.
```

Comment:
I do not think "least" is the correct word here.

Reply:
Or maybe just use the word "smallest"?